Iter15

DenisKhanov opened the pull request.

DenisKhanov committed:
Инкремент 14.
Немного изменил структуру репозитория (сохранение в память), для хранения всех ссылок пользователя.
Добавил 2 middleware аутентификации для публичных и приватных эндпоинтов.
В хендлере, сервисе и репозиториях добавились методы получения всех ссылок сохраненных пользователем.

DenisKhanov committed:
Инкремент 14.
Оказывается автотесты с ошибкой, ожидают не тот статус, поэтому пришлось исправить на возвращение статуса 204 на неправильность куки

DenisKhanov committed:
Инкремент 14.
Подправил нейминг констант авторизации, были через капс.

DenisKhanov committed:
Инкремент 15.
Первая простая версия асинхронного запуска удаления.

DenisKhanov committed:
Инкремент 15.
Сгенерировал новые мокки с учетом изменений.

DenisKhanov committed:
Инкремент 15.
Убрал лишний метод из сервиса. Исправил ошибку в двойном условии обработки ошибки асинхронного метода сервиса.

DenisKhanov committed:
Инкремент 15.
Подправил обработку ошибок в асинхронном методе сервиса и методе репозитория

edylcrm commented on internal/app/auth/auth.go:
```diff
@@ -0,0 +1,95 @@
+
+func generateUniqueID() uint32 {
+	rand.NewSource(time.Now().UnixNano())
+	for {
```
[LINT] а зачем тут цикл? Кажется можно удалить

DenisKhanov committed:
Инкремент 15.
Убрал лишний метод из сервиса. Исправил ошибку в двойном условии обработки ошибки асинхронного метода сервиса.
